src/reset.scss

```diff
@@ -0,0 +1,5 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
don't reset margins and paddings using * selector
will be better to specify needed separate tags, for which you need to reset them